VocableUITests/Screens/SettingsScreen.swift

```diff
@@ -14,6 +14,7 @@ class SettingsScreen: BaseScreen {
     let keyboardScreen = KeyboardScreen()
     
     let categoriesButton = XCUIApplication().collectionViews.staticTexts["Categories and Phrases"]
+    let timingAndSensitivityButton = XCUIApplication().collectionViews.staticTexts["Timing and Sensitivity"]
     let otherElements = XCUIApplication().collectionViews.cells.otherElements
     let cells = XCUIApplication().cells
     let settingsPageNextButton = XCUIApplication().buttons["bottomPagination.right_chevron"]
```

Comment:
```suggestion
    static let categoriesButton = XCUIApplication().collectionViews.staticTexts["Categories and Phrases"]
    static let timingAndSensitivityButton = XCUIApplication().collectionViews.staticTexts["Timing and Sensitivity"]
    static let otherElements = XCUIApplication().collectionViews.cells.otherElements
    static let cells = XCUIApplication().cells
    static let settingsPageNextButton = XCUIApplication().buttons["bottomPagination.right_chevron"]
```
I believe all of these can be static with Rudy's latest, which should be a nice ergonomic improvement. Same goes for the functions below 😄